README.md

```diff
@@ -16,4 +26,3 @@ API for NDLA "show and filter" service
 
```

Comment:
`Package and run locally` gir ingen mening med mindre man har tilgang til deploy-repoet (som kun vi internt har), så dette avsnittet synes jeg vi kan fjerne?